cmd/gitpods/main.go

```diff
@@ -20,12 +20,44 @@ func main() {
 		Usage:  "Runs gitpods on you local development machine",
 		Action: actionDev,
 		Flags: []cli.Flag{
-			cli.StringFlag{Name: "addr-ui", Usage: "The address to run the UI on", Value: ":3010"},
-			cli.StringFlag{Name: "addr-api", Usage: "The address to run the API on", Value: ":3020"},
-			cli.StringFlag{Name: "env", Usage: "Set the env gitpods runs in", Value: "development"},
-			cli.StringFlag{Name: "log-level", Usage: "The log level to filter logs with before printing", Value: "debug"},
-			cli.BoolFlag{Name: "setup", Usage: "Setup all dependencies needed for local development"},
-			cli.BoolFlag{Name: "watch,w", Usage: "Watch files in this project and rebuild binaries if something changes"},
+			cli.StringFlag{
+				Name:  "addr-ui",
+				Usage: "The address to run the UI on",
+				Value: ":3010",
+			},
+			cli.StringFlag{
+				Name:  "addr-api",
+				Usage: "The address to run the API on",
+				Value: ":3020",
+			},
+			cli.StringFlag{
+				Name:  "database-driver",
+				Usage: "The database driver to use: memory & postgres",
+				Value: "postgres",
+			},
+			cli.StringFlag{
+				Name:  "database-datasource",
+				Usage: "The database connection data",
+				Value: "postgres://postgres:postgres@localhost:5432?sslmode=disable",
+			},
+			cli.StringFlag{
+				Name:  "env",
+				Usage: "Set the env gitpods runs in",
+				Value: "development",
```

Comment:
It's more or less an artifact from the beginning. I did a switch on that to return the correct logger. Might be time to remove it because you can configure the logger without that.